src/Microsoft.Diagnostics.Monitoring.Egress/FileSystem/FileSystemEgressEndpointOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Monitoring.Egress.FileSystem
+{
+    internal class FileSystemEgressEndpointOptions
```

Comment:
Consider adding an option to create the file in the temp directory and then move it to the final destination to allow for file watching mechanisms to pick up completely written files.

Reply:
Done